<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverflowPaletteWindow</class>
 <widget class="QMainWindow" name="OverflowPaletteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1018</width>
    <height>705</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OverflowPaletteWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QTabWidget" name="MainTab">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="elideMode">
       <enum>Qt::TextElideMode::ElideRight</enum>
      </property>
      <widget class="QWidget" name="GameTab">
       <attribute name="title">
        <string>Game</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="GameConfigManager" name="ConfigManager" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(102, 102, 102);</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="OverflowPaletteGamePanelWidget" name="MainGamePanel" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="GameEditor">
       <attribute name="title">
        <string>Editor</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="GameEditorMainWidget" name="GameEditorMainPanelWidget" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1018</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OverflowPaletteGamePanelWidget</class>
   <extends>QWidget</extends>
   <header>OverflowPaletteGamePanelWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameConfigManager</class>
   <extends>QWidget</extends>
   <header>GameConfigManager.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameEditorMainWidget</class>
   <extends>QWidget</extends>
   <header>GameEditorMainWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
